docs/annotating.md

```diff
@@ -1,4 +1,52 @@
 ## Annotating datasets
 
+Storing data is one thing. Providing essential information about the
+data is the other. In order to allow later reuse of once-acquired data
+and to achieve reproducibility of scientific results we also need the
+means to of storing sufficient metadata.
+
+In the *NIX* [data model](./data_model.md) we allow annotation of data
+on several levels. It is possible to store global information, for
+example about the experimental session, and also to annotate the data
+itself. In the extreme it would be possible to annotate individual
+data points.
+
+We implemented the (*odml*)[http://g-node.github.io/python-odml] data
```

Comment:
Markdown linking is `[text](url)` not the other way around (I make this mistake all the time).